mixins/demo/localize-test-async.js

```diff
@@ -0,0 +1,72 @@
+import { html, LitElement } from 'lit-element/lit-element.js';
+import { LocalizeMixin } from '../../mixins/localize-mixin.js';
```

Comment:
I broke the demo components into separate static and async versions to simplify things.